src/power_grid_model_io/converters/vision_excel_converter.py

```diff
@@ -37,16 +37,21 @@ def __init__(
         self,
         source_file: Optional[Union[Path, str]] = None,
         language: str = "en",
+        terms_changed: Optional[dict] = None,
         mapping_file: Optional[Path] = None,
```

Comment:
to keep the parallel as for the `source_file` argument: 

```suggestion
        mapping_file: Optional[Union[Path, str]] = None,
```

Reply:
Unfortunately this will fail one of the legacy test

Reply:
why? a `Union` type should not impact anything because the input type `Path` remains valid

Reply:
I read the wrong document name xD